### PR TITLE
2nd part of the task

### PR DESCRIPTION
**Pull request by cybercubes**
1. Add "Country" form with button (the same as other forms), that will generate random int (from 1 to 10).
finalbuild

### CODE REVIEW

File: core/dist/dependencies.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+dependencies=
```
Comment (CrazyFlasher) [inline core/dist/dependencies.properties]: delete this file from repository (do not add and commit it)

File: core/dist/pom.xml.dependencies [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (CrazyFlasher) [inline core/dist/pom.xml.dependencies]: delete this file from repository (do not add and commit it)

File: examples/simpleMVC/dist/dependencies.properties [hunk 3]
```diff
@@ -0,0 +1 @@
+dependencies=C:/Users/Kirill/.m2/repository/com/domwires/core/0.9.9-SNAPSHOT/core-0.9.9-SNAPSHOT.swc,C:/Users/Kirill/.m2/repository/com/domwires/starlingApp/0.9.12-SNAPSHOT/starlingApp-0.9.12-SNAPSHOT.swc,C:/Users/Kirill/.m2/repository/com/gamua/starling/2.3.0/starling-2.3.0.swc,C:/Users/Kirill/.m2/repository/com/feathersui/feathers/3.1.1/feathers-3.1.1.swc,C:/Users/Kirill/.m2/repository/com/feathersui/MetalWorksMobileTheme/3.1.1/MetalWorksMobileTheme-3.1.1.swc
```
Comment (CrazyFlasher) [inline examples/simpleMVC/dist/dependencies.properties]: delete this file from repository (do not add and commit it)

File: examples/simpleMVC/dist/pom.xml.dependencies [hunk 4]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (CrazyFlasher) [inline examples/simpleMVC/dist/pom.xml.dependencies]: delete this file from repository (do not add and commit it)

File: examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as [hunk 5]
```diff
@@ -64,6 +68,11 @@ package com.domwires.example.simplemvc.view
 		{
 			firstNameLabel.text = model.firstName;
 		}
+		
+		private function CountryNameChanged(m:IMessage):void
```
Comment (CrazyFlasher) [inline examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as]: According Java and AS3 conventions, the first char of methods should be lowercase

**Review by CrazyFlasher — CHANGES_REQUESTED** [review]
Attach screenshot of final build

File: core/dist/dependencies.properties (hunk 1 above)
Comment (cybercubes) [reply core/dist/dependencies.properties]: deleted  core/dist/dependencies.properties

File: core/dist/pom.xml.dependencies (hunk 2 above)
Comment (cybercubes) [reply core/dist/pom.xml.dependencies]: deleted core/dist/pom.xml.dependencies

File: examples/simpleMVC/dist/dependencies.properties (hunk 3 above)
Comment (cybercubes) [reply examples/simpleMVC/dist/dependencies.properties]: deleted examples/simpleMVC/dist/dependencies.properties

File: examples/simpleMVC/dist/pom.xml.dependencies (hunk 4 above)
Comment (cybercubes) [reply examples/simpleMVC/dist/pom.xml.dependencies]: deleted examples/simpleMVC/dist/pom.xml.dependencies

File: examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as (hunk 5 above)
Comment (cybercubes) [reply examples/simpleMVC/src/com/domwires/example/simplemvc/view/AppView.as]: changed AppView.as methods in accordance with Java and AS3 conventions

**Pull request merged**